docs/resources/supported_operations.md

```diff
@@ -0,0 +1,1072 @@
+       ShuffleChannels                    Supported       Supported**      Not Supported     Supported       
+       Sign                               Supported       Supported**      Not Supported     Supported       
+       Sin                                Supported       Supported**      Not Supported     Supported       
+       Sinh                               Supported       Supported**      Not Supported     Supported****
+       SimplerNMS                         Supported       Supported**      Not Supported     Not Supported   
+       Slice                              Supported       Supported***     Supported         Not Supported   
+       SoftMax                            Supported       Supported***     Not Supported     Supported       
+       Softplus                           Supported       Supported**      Not Supported     Supported       
+       Softsign                           Supported       Supported**      Supported         Not Supported   
+       SpaceToDepth                       Not Supported   Supported**      Not Supported     Supported*     
+       SpatialTransformer                 Not Supported   Supported**      Not Supported     Not Supported   
+       Split                              Supported       Supported***     Supported         Supported       
+       Squeeze                            Supported       Supported**      Supported         Supported       
+       StridedSlice                       Supported       Supported**      Not Supported     Supported*     
+       Tan                                Supported       Supported**      Not Supported     Supported****
+       TensorIterator                     Not Supported   Supported        Supported         Supported       
+       Tile                               Supported**     Supported***     Not Supported     Supported       
+       TopK                               Supported       Supported**      Not Supported     Supported****
+       Unpooling                          Supported       Not Supported    Not Supported     Not Supported   
+       Unsqueeze                          Supported       Supported**      Supported         Supported       
+       Upsampling                         Supported       Not Supported    Not Supported     Not Supported   
+      =================================  ==============  ===============  ================  ==================
+      
+      |   `*` - support is limited to the specific parameters. Refer to "Known Layer Limitations" section for the device :doc:`from the list of supported <openvino_docs_OV_UG_supported_plugins_Supported_Devices>`.
+      |   `**` - support is implemented via :doc:`Extensibility mechanism <openvino_docs_Extensibility_UG_Intro>`.
+      |   `***` - supports NCDHW layout.
+      |   `****` - support is implemented via runtime reference.
+
+
+   .. tab-item:: Operations by Frameworks
```

Comment:
let us decouple `operations by framework` into a separate document `supported_framework_operations.md` just to avoid mixing our opset operations and frameworks' ones.

Reply:
The idea here is to aggregate information on operations in general in one place, possibly combining it with the IR and OpSets section in the future.
Currently, operations by framework are in a separate article but it is still not clear what the article presents exactly - are these ops supported by frameworks, or by these frontends in OV, or are these OV-supported layers pertaining to specific frameworks? 
Do you think we could keep these two tables in one article and provide a clear distinction between the two by proper descriptions? This way the information would be all in one place and clearly explained.

Reply:
I am leaning not to have long document and decouple framework operations support by FEs and opset operations support by the plugins.